Dockerfile

```diff
@@ -1,50 +1,35 @@
 FROM ruby:2.6.3
 
-LABEL Name=rails_api_base Version=1.0
+RUN apt-get update -qq && \
+    apt-get install -y build-essential openssl libssl-dev nodejs libpq-dev less vim nano libsasl2-dev
```

Comment:
just curious, what is the advantage of libpq-dev over postgresql-client ?

Reply:
Actually I changed it, because thoughtbot dockerfile was using it. After doing some research I found out that `libpq-dev` should have a smaller footprint than `postgresql-client`.
[Here](https://www.postgresql.org/download/linux/ubuntu/) you can check it out.